relay-dynamic-config/src/global.rs

```diff
@@ -167,6 +167,18 @@ pub struct Options {
     )]
     pub feedback_ingest_topic_rollout_rate: f32,
 
+    /// Flag for handling feedback and attachments in the same envelope. Temporary FF for fast-revert
+    /// (will remove after user feedback GA release).
```

Comment:
Without context, this comment is not clear to me what the flag should do. What do you think about something like the following?

```suggestion
    /// Kill switch to stop producing feedback items separately into Kafka. The topic is controlled by `feedback_ingest_topic_rollout_rate`.
```